libs/partners/huggingface/langchain_huggingface/llms/huggingface_pipeline.py

```diff
@@ -290,7 +291,8 @@ def _generate(
                         f"Got invalid task {self.pipeline.task}, "
                         f"currently only {VALID_TASKS} are supported"
                     )
-
+                if skip_prompt:
+                    text = text[len(batch_prompts[j]) :]
```

Comment:
could we document this somewhere? or add the kwarg to the function signature? otherwise how will people know they can use this?

Reply:
done